### PR TITLE
Format_disk: modify dmesg command for rhel610 guest on ppc

### PR DESCRIPTION
**Pull request by MiriamDeng**
ID:2177608
Signed-off-by: MiriamDeng <mdeng@redhat.com>

### CODE REVIEW

**Comment by MiriamDeng** [comment]
JOB LOG    : /root/avocado/job-results/job-2023-03-13T02.09-75267dc/job.log
 (1/1) Host_RHEL.m7.u9.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.6.10.ppc64.io-github-autotest-qemu.format_disk: PASS (148.69 s)

**Comment by MiriamDeng** [comment]
@aliang123 @maxujun @PaulYuuu could you help this patch ? Thanks a lot.

File: qemu/tests/format_disk.py [hunk 1]
```diff
@@ -33,7 +33,8 @@ def run(test, params, env):
     os_type = params["os_type"]
 
     if os_type == 'linux':
-        session.cmd("dmesg -C")
+        dmesg_cmd = params.get("dmesg_cmd")
```
Comment (aliang123) [inline qemu/tests/format_disk.py:36]: What about dmesg_cmd = params.get("dmesg_cmd", "dmesg -C")？

Then in your format_disk.cfg, you just need to set:
   RHEL.6:
        fs_type = ext4
        dmesg_cmd = "dmesg -c"

Comment (MiriamDeng) [reply qemu/tests/format_disk.py]: Thanks @aliang123 , updated ! please help to review again :)

Comment (MiriamDeng) [reply qemu/tests/format_disk.py]: Hi @aliang123 corrected, thanks !

**Comment by aliang123** [comment]
Acked-by: Aihua Liang <aliang@redhat.com>

**Comment by maxujun** [comment]
LGTM, Acked.

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**